hash/ibm5170_hdd.xml

```diff
@@ -32,13 +32,13 @@ license:CC0-1.0
 		</part>
 	</software>
 
-	<software name="freedos13rc4">
-		<description>FreeDOS 1.3 Release Candidate 4</description>
-		<year>2021</year>
+	<software name="freedos13">
+		<description>FreeDOS 1.3</description>
+		<year>2022</year>
 		<publisher>The FreeDOS Project</publisher>
 		<part name="hdd" interface="ide_hdd">
 			<diskarea name="harddriv">
-				<disk name="freedos13rc4" sha1="bb81d708746f4e5cfd4207af598cd7691a8e33bf" writeable="yes"/>
+				<disk name="freedos13" sha1="72eef7bf0ad53b8a240ce05244173df54db923f0" writeable="yes"/>
 			</diskarea>
 		</part>
 	</software>
```

Comment:
Note: FreeDOS 1.3 (at least) has a out-the-box bug around CPU detection (which comes from the `vinfo /m` calls), so for example it will wrongly goto as 80186 in `ct486`.
![0003](https://github.com/user-attachments/assets/d6236f8d-7cb5-4632-929a-828d0cd55f6d)
https://gitlab.com/FreeDOS/issue-reporting/-/issues/26

I guess this is worth mentioning in a`<notes>` line for this HDD entry, giving an hint to the end user about it.

Reply:
I could work around it by editing fdauto.bat to always go into the 286/386 routines (this disk image is made for ibm5170, after all), but that'd both require me to make a new inp (install from scratch, then edit), and I'd feel kind of dirty for making a modified OS image.

At least a notes line wouldn't involve either of those...

Reply:
Adding the notes is easy enough, I think, but could you give me a clue to where in the MAME UI it's supposed to display? I can't find it at all.

Reply:
If I'm not mistaken, the info usage elements should be displayed in UI.

The notes field is just for notes. We use it mostly for documenting things that are broken with the emulation.

Reply:
The trouble with info usage is that it doesn't allow for multiline commentary afaik, so long lines will go way after any conventional file width limit. fwiw frontend has no limit ...
![image](https://github.com/user-attachments/assets/f09a2cd3-faf0-4ceb-ad67-2483a66d83da)

Reply:
I'm not even sure it's necessary. It's not really MAME's job to point out all software bugs, and this one doesn't cause FreeDOS to be unusable.